client/src/components/Navbar.jsx

```diff
@@ -1,35 +1,179 @@
+              transformOrigin={{
+                vertical: "top",
+                horizontal: "left",
+              }}
+              open={Boolean(anchorElNav)}
+              onClose={handleCloseNavMenu}
+              sx={{
+                display: { xs: "block", md: "none" },
+                mt: 5,
+              }}
+              PaperProps={{
+                sx: {
+                  backgroundColor: "brown.100",
+                  width: 300,
+                  p: 1,
+                },
+              }}
+            >
+              {pageNames.map((name, i) => (
+                <MenuItem
+                  key={name}
+                  onClick={handleCloseNavMenu}
+                  bgcolor="black"
+                >
+                  <Link href={pageLinks[i]} underline="none" color="gray.800">
+                    {name}
+                  </Link>
+                </MenuItem>
+              ))}
+            </Popover>
+            <Link
+              href="/"
+              title="Internship Tracker Home Page"
+              sx={{ display: { xs: "flex", md: "none" }, mr: 1 }}
+            >
+              <img src={LogoPath} alt="Internship Tracker Logo" height="32" />
+            </Link>
+          </Box>
+          <Box sx={{ flexGrow: 1, display: { xs: "none", md: "flex" } }}>
+            {pageNames.map((name, i) => (
+              <Button
+                key={name}
+                onClick={handleCloseNavMenu}
+                href={pageLinks[i]}
+                sx={{
+                  ml: 3,
+                  display: "block",
+                }}
+              >
+                <Typography variant="navText">{name}</Typography>
+              </Button>
+            ))}
+          </Box>
+          {authUser ? (
+            <Box>
+              <Button sx={{ p: 0 }}>
+                <img src={TrackerIcon} alt="tracker" />
+              </Button>
+              <Button sx={{ p: 0 }}>
+                <img src={AccountIcon} alt="account" />
+              </Button>
+            </Box>
+          ) : (
+            <Box>
+              <Button>
+                <Typography variant="navText">Sign In</Typography>
+              </Button>
+              <Button
+                variant="rounded"
+                color="thai"
+                sx={{
+                  ml: { sx: 1, sm: 2 },
+                  padding: { sx: ".3rem .6rem", sm: ".6rem 1.3rem" },
+                }}
+              >
+                <Typography
+                  variant="navText"
+                  color="white"
+                  sx={{ display: { xs: "none", sm: "flex" } }}
+                >
+                  Create an account
+                </Typography>
+                <Typography
+                  variant="navText"
+                  color="white"
+                  sx={{ display: { xs: "flex", sm: "none" } }}
+                >
+                  Sign Up
+                </Typography>
+              </Button>
+            </Box>
```

Comment:
Can we remove the account buttons or comment these out for now? Have we decided as a team to include creating accounts/registration in our first milestone? My concern lies in if this project is released in two phases where users are able to use the application to search for internships, even if we have accounts, there is no functional reason to collect student information at that time.

If we collect their emails, pws, etc, theres overhead and extra features that need to be included, but signing in also offers no benefit to the ability to search for internships. In addition we would also have to pull in work to also set up password resets

Reply:
we can discuss this in the discord thread you have created...for now I've set the default value of authUser to be true